packages/baseui-codemods/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org
```

Comment:
Was it committed intentionally?

Reply:
yeah - i couldn't publish until this was specified. it would try to publish to private npm. figure this will help anyone else who tries to publish in future.